components/Ink/src/private/MDCInkLayer.m

```diff
@@ -187,7 +188,7 @@ - (void)endAnimationAtPoint:(CGPoint)point {
   fadeOutAnim.keyPath = MDCInkLayerOpacityString;
   fadeOutAnim.fromValue = @(opacity);
   fadeOutAnim.toValue = @0;
-  fadeOutAnim.duration = MDCInkLayerStartScalePositionDuration;
+  fadeOutAnim.duration = MDCInkLayerEndFadeOutDuration;
```

Comment:
Break this out to its own change please.

Reply:
Done. Moved to https://github.com/material-components/material-components-ios/pull/2576